test(Path): fix tests

shivam1646 opened the pull request:
resolves #693

divyanshu013 commented on packages/blade/src/components/Icons/_Svg/Path/__tests__/Path.web.test.tsx:
```diff
@@ -25,7 +26,7 @@ describe('<Path />', () => {
   it('should not accept dashed props', () => {
     // @ts-expect-error no dashed props are accepted
-    const { container } = renderWithTheme(<Path fill-opacity={0} d="" />);
+    const { container } = renderWithTheme(<Path fill-opacity={0} d="" is="x3d" />);
```
How does this resolve the issue?

shivam1646 replied:
So in the tests, the error indicates that browser does not recognise `path` as a valid dom tag. Using `is="x3d"` in the tests, tells browser to recognise this element as a valid SVG tag.

divyanshu013 commented on packages/blade/src/components/Icons/_Svg/Path/__tests__/__snapshots__/Path.web.test.tsx.snap:
```diff
@@ -18,16 +19,17 @@ exports[`<Path /> should render html path component 1`] = `
-      stroke-width="2px"
+      strokeLinecap="square"
+      strokeLinejoin="bevel"
+      strokeWidth="2px"
```
This change would actually break the svgs because camel cased props work only in React, not in the actual browser.

shivam1646 replied:
This change affects only the test file - `Path.web.test.tsx` where the error is thrown.

divyanshu013 reviewed (changes requested):
I think this change would end up breaking things instead because of how it affects the props. The issue is probably something related to how jest is understanding `path`.

shivam1646 commented:
@divyanshu013 
I misunderstood the issue earlier.

Actually, one of the test cases `'should not accept dashed props'` in `Path.web.test.tsx` uses `path` tag without a parent `svg` tag which leads to browser not able to recongnize the `path` tag.

Added the fix accordingly and reverted the previous changes. Sorry for the confusion.

divyanshu013 commented:
Awesome, this seems like the correct fix 🎉

divyanshu013 reviewed (approved):
Looks good to me.
